extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -222,662 +223,716 @@ For `region` bounding volumes, the coordinates are interpreted in geographic coo
+| 16-23 | Binary byte length | `uint64_t` | The length of the buffer (or 0 if the buffer does not exist) including any padding. |
+
+Each chunk must be padded so it ends on an 8-byte boundary:
+
+* The JSON chunk must be padded at the end with spaces (ASCII `' '` = 0x20)
+* If it exists, the binary chunk must be padded at the end with NUL bytes (`\x00` = 0x00)
+
+The subtree JSON describes where the availability information for a single subtree is stored. Availability bitstreams are stored in buffers and accessed through buffer views.
 
 ### Buffers and Buffer Views
 
-A **buffer** is an array of binary data used for storing data. For subtree binary files, one buffer can be stored within the binary file. In all other cases, the binary file is assumed to be an external resource specified by the `uri` property. Each buffer has a `byteLength` describing the size of the data, including any padding (for subtree binary files)
+A **buffer** is a binary blob. A single buffer can be stored within the binary chunk of a subtree file. In all other cases, the binary file is assumed to be an external resource specified by the `uri` property. Each buffer has a `byteLength` describing the size of the data, including any padding (for subtree binary files)
 
-A **buffer view** is a contiguous subarray of a buffer. A buffer view's `buffer` property is an integer index to identify the buffer. A buffer must have a `byteOffset` to describe the position within the buffer. A buffer also must have a `byteLength` length of the subarray, not including padding. There may be multiple buffer views within a single buffer.
+A **buffer view** is a contiguous subset of a buffer. A buffer view's `buffer` property is an integer index to identify the buffer. A buffer view has a `byteOffset` and a `byteLength` to describe the range of bytes within the buffer. The `byteLength` does not include any padding. There may be multiple buffer views referencing a single buffer.
 
 For efficient memory access, the `byteOffset` of a buffer view must be aligned to a multiple of 8 bytes.
 
-```jsonc
+```json
 {
   "buffers": [
     {
       "name": "Internal Buffer",
-      "byteLength": 80
+      "byteLength": 16
     },
     {
       "name": "External Buffer",
       "uri": "external.bin",
-      "byteLength": 64
+      "byteLength": 32
     }
   ],
-  "bufferViews" [
+  "bufferViews": [
     {
-      "name": "Tile Availability",
       "buffer": 0,
       "byteOffset": 0,
-      "byteLength": 3
+      "byteLength": 11
     },
     {
-      "name": "Content Availability",
-      "buffer": 0,
-      "byteOffset": 8,
-      "byteLength": 3
-    },
-    {
-      "name": "Subtree Availability",
       "buffer": 1,
       "byteOffset": 0,
-      "byteLength": 64
+      "byteLength": 32
     }
-  ]
+  ],
+  "tileAvailability": {
+    "constant": 1,
+  },
+  "contentAvailability": {
+    "bufferView": 0
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 1
+  }
```

Comment:
It would be good to summarize this example a bit to drive home the points, especially why `constant: 1` is being used - because all tiles exist - in contrast to content availability where only some content resources exist.